lib/ui/recommendation.js

```diff
@@ -19,16 +19,21 @@ function Recommendation(opts) {
   this.io = opts.io;
   this.objectUtils = opts.objectUtils;
   this.RecommendationRow = opts.RecommendationRow;
+  this.transforms = {
+    tld: opts.tldTransform,
+    wikipedia: opts.wikipediaTransform,
+    movie: opts.movieTransform
+  };
 
   this.el = null;
-  this.timeout = null;
+  this.hideTimeout = null;
```

Comment:
Add a comment in the final PR that this name change was just to clarify the purpose of the timeout.